_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,72 @@
+// ----------------------------
```

Comment:
This file's name does not match what we were expecting in our main style compile - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#table-of-contents. Its purpose is really reset a bunch of stuff for our preferences. I'd recommend renaming it to ``sass/base/_reset.scss``